modules/cache/main.tf

```diff
@@ -0,0 +1,66 @@
+data "aws_route53_zone" "public" {
+  name = var.zone_name
+}
+
+module "redis" {
+  count = var.cache_store == "redis" ? 1 : 0
+
+  source  = "cloudposse/elasticache-redis/aws"
+  version = "0.30.0"
+
+  name = "redis"
```

Comment:
How about <project>-redis?

Reply:
it has namespace below and as a result, it will generate <project>-redis.